packages/exampleReactNative/README.md

```diff
@@ -0,0 +1,33 @@
+# Running
+
+Make sure Xcode (iOS) and Android Studio (Android) are installed. Also make sure [CocoaPods](https://cocoapods.org/) and a JDK are installed.
+
+1. `cd exampleReactNative/../../` i.e. the project root
```

Comment:
Would really remove this, it just seems very complicated. If we had the `yarn example:rn:packager` and `yarn example:rn:ios` (and friend), we cover everything.

Reply:
Agree it was over complicated. Left notes about having stuff installed and choosing the simulator because feel that people might get stuck at those points.